### PR TITLE
Friendlier PR comment, helps with #105

### PR DESCRIPTION
**Pull request by laughinghan**
https://github.com/cla-assistant/cla-assistant/issues/105#issuecomment-194215536

If you haven't signed the CLA for a project, it's very likely your first
PR ever to this project. And the first thing that happens, within
seconds, is a comment by this bot with an imperative to jump through
another hoop? Oops, I think we'd like to at least show appreciation for
their effort.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cla-assistant/cla-assistant/pull/110" target="_blank">ReviewNinja

File: src/server/services/pullRequest.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var commentText = function (signed, badgeUrl, claUrl, user_map) {
 			text += '<br/>:x: ' + signee;
 		});
 	} else {
-		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>';
+		text = '[![CLA assistant check](' + badgeUrl + ')](' + claUrl + ') <br/>Thank you for your submission, we really appreciate your work. Like many open source projects, before we can accept your contribution, we first ask that you sign our [Contributor License Agreement](' + claUrl + ').<br/>';
```
Comment (KharitonOff) [inline src/server/services/pullRequest.js]: It looks much better now. 
I would only skip the "Like many open source projects" part
@dfarr , @thojansen  : what do you think?

Comment (laughinghan) [inline src/server/services/pullRequest.js]: As someone who has a knee-jerk negative reaction to EULAs and the like, I
think a gentle reminder that however distasteful this is standard practice
would at least somewhat mollify me.

I'm certainly open to other phrasings, maybe more matter-of-fact like "As
an open source project, ..." ('cos, it is/should be standard practice,
right)?

Han

Comment (thojansen) [inline src/server/services/pullRequest.js]: I like it. Could we rephrase it to sth like .."In order to obtain necessary ownership/ grants of rights, before we can accept.."?

Comment (dfarr) [inline src/server/services/pullRequest.js]: > before we can accept your contribution, we first ask that you sign...

The "first" is redundant here. Instead, simply "before we can accept your contribution, we ask that you sign..."

Comment (casche) [inline src/server/services/pullRequest.js]: > Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](' + claUrl + ') before we can accept your contribution.

I took @dfarr's suggestion and re-arranged things a bit. @dfarr what do you think of this version?

Comment (dfarr) [inline src/server/services/pullRequest.js]: :+1:

Comment (laughinghan) [inline src/server/services/pullRequest.js]: Updated

**Comment by thojansen** [comment]
:+1:
